File: Python/Remove Element.py
class Solution:
    def removeElement(self, A, elem):
        if not A:
            return None
        length = len(A)
        count = 0
        for i in range(length):
            if A[i] == elem:
                count = count + 1
            else:
                A[i-count] = A[i]
        return length - count

File: Java/src/Remove Element.java
public class Solution {
public int removeElement(int[] A, int elem) {
int length = A.length;
int count = 0;
for(int i = 0 ; i < length; i++){
if(A[i] == elem){
count++ ;
}
else{
A[i-count] = A[i];
}
}
return length-count;
}
}